diff -Naur gmp-6.1.0/configfsf.sub aplus-gmp-6.1.0/configfsf.sub
--- gmp-6.1.0/configfsf.sub	2015-11-01 16:19:48.000000000 +0100
+++ aplus-gmp-6.1.0/configfsf.sub	2022-12-06 02:17:06.046838064 +0100
@@ -1374,7 +1374,7 @@
 	# The portable systems comes first.
 	# Each alternative MUST END IN A *, to match a version number.
 	# -sysv* is not here because it comes later, after sysvr4.
-	-gnu* | -bsd* | -mach* | -minix* | -genix* | -ultrix* | -irix* \
+	-gnu* | -bsd* | -mach* | -minix* | -genix* | -ultrix* | -irix* | -aplus* \
 	      | -*vms* | -sco* | -esix* | -isc* | -aix* | -cnk* | -sunos | -sunos[34]*\
 	      | -hpux* | -unos* | -osf* | -luna* | -dgux* | -auroraux* | -solaris* \
 	      | -sym* | -kopensolaris* | -plan9* \
