--- libwebp/config.sub	2022-09-11 18:12:51.835738274 +0200
+++ aplus-libwebp/config.sub	2022-09-11 18:09:46.743771904 +0200
@@ -1718,7 +1718,7 @@
 	# Now accept the basic system types.
 	# The portable systems comes first.
 	# Each alternative MUST end in a * to match a version number.
-	gnu* | android* | bsd* | mach* | minix* | genix* | ultrix* | irix* \
+	gnu* | android* | bsd* | mach* | minix* | genix* | ultrix* | irix* | aplus* \
 	     | *vms* | esix* | aix* | cnk* | sunos | sunos[34]* \
 	     | hpux* | unos* | osf* | luna* | dgux* | auroraux* | solaris* \
 	     | sym* |  plan9* | psp* | sim* | xray* | os68k* | v88r* \
